shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1660,7 +1659,7 @@ public void reset() {
 
   // Must match SemanticsFlag in semantics.dart
   // https://github.com/flutter/engine/blob/master/lib/ui/semantics.dart
-  private enum Flag {
+  public enum Flag {
```

Comment:
I'm not sure of the implications

Reply:
```suggestion
  /* Package */ enum Flag {
```

Reply:
doesn't need to be public if you just want it in a test (which is in the same package and default visibility is fine)